tsl/src/dist_util.c

```diff
@@ -234,6 +234,23 @@ dist_util_remote_chunk_info(PG_FUNCTION_ARGS)
 	return dist_util_remote_srf_query(fcinfo, node_name, query_str->data);
 }
 
+Datum
+dist_util_remote_compressed_chunk_info(PG_FUNCTION_ARGS)
+{
+	char *node_name;
+	StringInfo query_str = makeStringInfo();
+	/* Strict function */
+	Name schema_name = PG_GETARG_NAME(1);
+	Name table_name = PG_GETARG_NAME(2);
+	Assert(!PG_ARGISNULL(0) && !PG_ARGISNULL(1) && !PG_ARGISNULL(2));
```

Comment:
can you make this an actual check strict checks only apply when calling functions from sql so its better to have those checks in place and also PG_NARGS

Reply:
Can you elaborate what you are asking for?

Reply:
change the assert into if and check PG_NARGS == 3

Reply:
I would like to understand the issue better. This is called via SQL/Pgsql and declared as strict. Are you saying that we are missing the strictness check via PG here? Isn't that the case only if we call via C?

Reply:
I think what @svenklemm is saying is that if the function is marked `STRICT` it won't be called if any of the arguments are NULL, so `PG_NARGS` must be equal to 3.